model/task/task.go

```diff
@@ -2225,66 +2226,41 @@ func (t *Task) hasUnattainableDependency() bool {
 	return false
 }
 
-// AbortBuild sets the abort flag on all tasks associated with the build which are in an abortable
-// state
-func AbortBuild(buildId string, reason AbortInfo) error {
+// AbortBuildTasks sets the abort flag on all tasks associated with the build which are in an abortable
+func AbortBuildTasks(buildId string, reason AbortInfo) error {
 	q := bson.M{
 		BuildIdKey: buildId,
 		StatusKey:  bson.M{"$in": evergreen.TaskInProgressStatuses},
 	}
 	if reason.TaskID != "" {
 		q[IdKey] = bson.M{"$ne": reason.TaskID}
 	}
-	ids, err := findAllTaskIDs(db.Query(q))
-	if err != nil {
-		return errors.Wrapf(err, "finding tasks to abort from build '%s'", buildId)
-	}
-	if len(ids) == 0 {
-		grip.Info(message.Fields{
-			"message": "no tasks aborted for build",
-			"buildId": buildId,
-		})
-		return nil
-	}
-
-	_, err = UpdateAll(
-		bson.M{IdKey: bson.M{"$in": ids}},
-		bson.M{"$set": bson.M{
-			AbortedKey:   true,
-			AbortInfoKey: reason,
-		}},
-	)
-	if err != nil {
-		return errors.Wrapf(err, "setting aborted statuses for tasks in build '%s'", buildId)
-	}
-
-	event.LogManyTaskAbortRequests(ids, reason.User)
-
-	return nil
+	return abortTasksByQuery(q, reason)
```

Comment:
Can we wrap the error here to make it clear it is being done for a build? `return errors.Wrapf(err, "setting aborted statuses for tasks in build '%s'", buildId)`